src/transformStateWithClones.js

```diff
@@ -8,6 +8,37 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
```

Comment:
giving your vars abstract names is awesome, but not this much, it may confuse other reviewers
generally, avoid giving names containing `result`, `res`, `arr`, `st`, instead do something like this – `arrayOfStates`, etc.